input/includes/get-gaps-caregap-usage-example.md

```diff
@@ -592,7 +592,7 @@ Content-Type: application/fhir+json;charset=UTF-8
         "id": "gaps-organization-reportingvendor",
         "meta": {
           "profile": [
-            "http://hl7.org/fhir/us/davinci-deqm/StructureDefinition/organization-deqm"
+            "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-organization"
```

Comment:
Related to https://jira.hl7.org/browse/FHIR-37898 which removed the DEQM Organization profile